frontend/app/admin/advertisement/create/page.tsx

```diff
@@ -0,0 +1,257 @@
+'use client'
+
+import {
+  Box,
+  FormControl,
+  Heading,
+  Stack,
+  FormLabel,
+  Input,
+  useToast,
+  Button,
+  Flex,
+  Text,
+  FormErrorMessage,
+  RadioGroup,
+  Radio,
+  HStack,
+} from '@chakra-ui/react'
+import React, { useState, FormEvent } from 'react'
+import { THEME_COLOR } from '../../../../constants'
+
+import { advertisementSchema } from '../../../../../common/zod'
+
+export default function CreateAdvertisementPage() {
+  const toast = useToast()
+  const [name, setName] = useState<string>('')
+  const [url, setUrl] = useState<string>('')
+  const [author, setAuthor] = useState<string>('')
+  const [showStatus, setShowStatus] = useState<'show' | 'notShow'>('notShow')
+  // TODO S3が入るようになってから広告画像保存できるように修正
+  const [imageUrl, setImageUrl] = useState<string>(
+    'https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.yuta-u.com%2Fprograming%2Freact-js-for-bigginer&psig=AOvVaw0l3T-LRKu7TksbHV0Vgp7O&ust=1707522385982000&source=images&cd=vfe&opi=89978449&ved=0CBIQjRxqFwoTCJiXs9z2nIQDFQAAAAAdAAAAABAE',
+  )
+  const [startFrom, setStartFrom] = useState<Date>(new Date())
+  const [endAt, setEndAt] = useState<Date>(new Date())
+  const [errors, setErrors] = useState<
+    {
+      message: string
+      path: string[]
+      type: string
+    }[]
+  >([])
+  const handleSubmit = async (event: FormEvent) => {
+    try {
+      const formData = {
+        name,
+        url,
+        author,
+        showStatus,
+        imageUrl,
+        startFrom,
+        endAt,
+      }
+
+      const validation = advertisementSchema.parse(formData)
+      console.log('validation', validation)
+
+      setShowStatus('show')
+
+      event.preventDefault()
+
+      const response = await fetch(
+        `${process.env.NEXT_PUBLIC_BACKEND_URL}/advertisement`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            name,
+            url,
+            author,
+            isShow: showStatus === 'show' ? true : false,
+            imageUrl,
+            startFrom,
+            endAt,
+          }),
+        },
+      )
+
+      const data = await response.json()
+
+      if (response.ok) {
+        toast({
+          title: data.message,
+          status: 'success',
+          position: 'top',
+          duration: 3000,
+        })
+      } else {
+        toast({
+          title: data.message,
+          status: 'error',
+          position: 'top',
+          duration: 3000,
+        })
+      }
+      setShowStatus('notShow')
+    } catch (e) {
+      if (e.issues) {
+        console.log('e', e.issues)
+        setErrors(e.issues)
+        return
+      }
+      toast({
+        title: 'エラーが発生しました',
+        status: 'error',
+        position: 'top',
+        duration: 3000,
+      })
+    }
+
```

Comment:
@preticure 
```suggestion
    }
}
```

で閉じてもらっていいですか？